pkg/actions/gvr.go

```diff
@@ -34,3 +37,12 @@ func GetGroupVersionResource(gr schema.GroupVersionResource, discovery discovery
 
 	return &gvr, nil
 }
+
+func InitializeAPIGroupRes(discovery discovery.DiscoveryInterface) error {
```

Comment:
do we need a separate function for this?

Reply:
we need this for unit tests to have the apiGroupRes, we need to call it from every test

Reply:
Any reason we need to call it from every test ? I would *like* this fix to not have to modify all the tests.

Reply:
Improved according to comments